### DISCUSSION TITLE
Error with i18next::translator: missingKey

### DISCUSSION THREAD

**Discussion by olga-gchak** [i18next/i18next#1696] [Q&A] [answered]
Hello!
Can you help me, please.

I use i18next to tranlation text from back-end.

But in first render of page I have this error:
![Screenshot_12](https://user-images.githubusercontent.com/67831961/143064316-03609b7c-94c1-494d-991e-bf22e1050a17.png)
 It's only part of text, all another text is transleted.
 
 When I click to any button everything fix.

**Comment by jamuhl** [comment, marked as answer]
Depending on UI framework used the cause is timing - loading translations from backend happens asynchronous -> in your case the first render seems to happen before i18next loaded the translations...

for eg. react https://react.i18next.com/latest/usetranslation-hook solves this...

for other frameworks you might use the init or load callback to make sure the translations were loaded: https://www.i18next.com/overview/api#loadnamespaces

**Reply by olga-gchak** [reply]
Thaks)
